app/src/main/java/org/openobservatory/ooniprobe/activity/BrowserActivity.java

```diff
@@ -148,6 +150,10 @@ public void onClick(View view) {
         }
     }
 
+    @Override
+    protected void attachBaseContext(Context newBase) {
+        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
+    }
 
```

Comment:
Okay, here you've basically added support for calligraphy inside the browser. I believe small changes like this can be submitted as quick-review pull requests that we can merge. Then, as I was saying on the iOS repository, we can use github's feature that we can merge master back into the PR and we get these small fixes in the branch anyway.

Reply:
This should be attached to every class to use the same font. Now the browser didn't need it because we have no labels, but better adding it

Reply:
Yes, I agree. Thanks for explaining. (As I said, I think things like this can be pushed as separate hotfix PRs.)